blivet/udev.py

```diff
@@ -286,6 +286,29 @@ def device_is_loop(info):
     return (device_get_name(info).startswith("loop") and
             os.path.isdir("%s/loop" % device_get_sysfs_path(info)))
 
+def device_is_realdisk(info):
```

Comment:
You should change `DeviceTree.udevDeviceIsDisk` to call this function as well so the code isn't in two places.